bio/varlociraptor/call-variants/wrapper.py

```
@@ -0,0 +1,15 @@
+import subprocess as sp
+
+log = snakemake.log_fmt_shell(stdout=False, stderr=True)
+
+obs = " ".join(f"{sample}={path}" for sample, path in zip(snakemake.params.samples, snakemake.input.observations))
```

Ruff

Undefined name `snakemake`

(F821)